src/Uno.UI/UI/Xaml/VisualStateGroup.cs

```diff
@@ -459,30 +459,31 @@ private VisualTransition FindTransition(string oldStateName, string newStateName
 			// The most specific transition wins (i.e. with matching From and To),
 			// then we validate for transitions that have only From or To defined which match.
 
+			var transitions = Transitions;
```

Comment:
Can it be tested?

Reply:
As usual with animations it would be difficult and most probably flaky on the CI :/